aws-cli-2.yaml

```diff
@@ -16,6 +16,12 @@ package:
       - aws-cli=${{package.full-version}}
     runtime:
       - groff-base
+      - libbz2-1
```

Comment:
I actually don't see anything in https://github.com/wolfi-dev/os/issues/40833 which requires the package `libbz2-1`. As far as I can tell that only provides `libbz2.so.1` - https://apk.dag.dev/https/packages.wolfi.dev/os/x86_64/libbz2-1-1.0.8-r10.apk@sha1:8ced019ea56cb2b2fd46174624cf38c1446c2c6e/.PKGINFO

Reply:
We need `libbz2-1` because of:

```
2025/03/03 23:23:29 INFO FAIL[ldd-check]: /usr/lib/aws-cli/lib-dynload/_bz2.cpython-311-x86_64-linux-gnu.so: missing libbz2.so.1
```

This is one of the dynamic Python modules that aws-cli-2 builds, and it requires the library.